apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/GenesisImporter.java

```diff
@@ -99,6 +113,10 @@ public GenesisImporter(
                 propertiesHolder.getIntProperty(PUBLIC_KEY_NUMBER_TOTAL_PROPERTY_NAME);
         this.balanceNumberTotal =
                 propertiesHolder.getIntProperty(BALANCE_NUMBER_TOTAL_PROPERTY_NAME);
+        this.pkTable = Objects.requireNonNull(pkTable, "pkTable is NULL");
+        this.genesisPublicKeyTable = Objects.requireNonNull(genesisPublicKeyTable, "genesisPublicKeyTable is NULL");
+        this.accountGuaranteedBalanceTable = Objects.requireNonNull(accountGuaranteedBalanceTable, "accountGuaranteedBalanceTable is NULL");
```

Comment:
Account table is also mandatory for cleanup.

Reply:
Done, added accountTable